vllm/model_executor/layers/quantization/awq_marlin.py

```diff
@@ -36,13 +38,18 @@ class AWQMarlinConfig(QuantizationConfig):
         8: scalar_types.uint8,
     }
 
-    def __init__(self, weight_bits: int, group_size: int, has_zp: bool,
-                 lm_head_quantized: bool) -> None:
+    def __init__(self,
+                 weight_bits: int,
+                 group_size: int,
+                 zero_point: bool,
+                 lm_head_quantized: bool,
+                 modules_to_not_convert: Optional[List[str]] = None) -> None:
         self.pack_factor = 32 // weight_bits  # packed into int32
         self.group_size = group_size
-        self.has_zp = has_zp
+        self.zero_point = zero_point
```

Comment:
nit: `zero_point` is better than `zp` but I think `has_zero_point` would be even better since then you know it's a boolean flag and not the actual zero point

Reply:
Here I'm choosing to just follow awq.py and the AWQ config naming to keep consistency